core/services/ocr2/plugins/functions/plugin.go

```diff
@@ -104,6 +104,7 @@ func NewFunctionsServices(functionsOracleArgs, thresholdOracleArgs, s4OracleArgs
 		conf.Job,
 		bridgeAccessor,
 		pluginORM,
+		s4ORM,
```

Comment:
Why not pass the same Storage object that GatewayConnector receives, instead of the ORM?

Reply:
Yeah I think this is right, will change

Reply:
Updated.